spec/github_helper_spec.rb

```diff
@@ -2,6 +2,30 @@
 require 'webmock/rspec'
 
 describe Fastlane::Helper::GithubHelper do
+  describe '#initialize' do
+    let(:client) do
+      instance_double(
+        Octokit::Client,
+        user: instance_double('User', name: 'test'),
+        'auto_paginate=': nil
+      )
+    end
+
+    before do
+      allow(Octokit::Client).to receive(:new).and_return(client)
+    end
+
+    it 'with the correct github_token' do
+      expect(described_class).to receive(:new).with(github_token: 'GITHUB_TOKEN')
+      described_class.new(github_token: 'GITHUB_TOKEN')
+    end
+
+    it 'Octokit client receives the correct github_token' do
+      expect(Octokit::Client).to receive(:new).with(access_token: 'GITHUB_TOKEN')
+      described_class.new(github_token: 'GITHUB_TOKEN')
```

Comment:
May I suggest to use a different dummy token? One that is less likely to be used by accident?

I could totally see us doing the easy mistake of a wrong implementation at some point (e.g. with a future change), that would explicitly use the string  `'GITHUB_TOKEN'` somewhere in our helper implementation (perhaps where we meant `ENV['GITHUB_TOKEN']` or something like that)… while we shouldn't use this string explicitly anymore after your changes (except in the `ConfigItem`'s `env_name`, but not anywhere else), so it would be nice if the unit tests would not unfortunately use that likely string as a dummy token.

```suggestion
      described_class.new(github_token: 'Fake-GitHubToken-123')
```

Reply:
That is indeed a good suggestion, I was using that `GITHUB_TOKEN` key by habit in all places.  Thanks!